### PR TITLE
Added Yowie as a playable species and Bogdan Accent

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Add's Yowie, a sasquatch-like roundstart species to the game

## Why / Balance
I wanted to make a race thats a bit unique in regards to chemicals and clothing, and also wanted a Big Lez Show reference in here

## Technical details
Added Yowie as a species (Doesn't Roll for ghost roles)
Added Bogan as an accent (Currently only available for Yowie, I feel like its better as a species unique accent)

## Media
image
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
I can't for the life of me get the marking system working, so this is the best current solution I have now. The right foot will always render above the left like this, but if people want multicoloured feet then they deserve 

**Changelog**
:cl:
- add: Added Yowie as a new roundstart species with the Bogan accent

## Release Notes

- **New Features**
  - Introduced the "Yowie" species with unique characteristics and abilities, including a new entity definition and damage modifiers.
  - Added "Bogan Accent" functionality for message modification, enhancing communication within the game.
  - New inventory template for Yowie characters with structured slots for various items.
  - Localization support for Yowie names and markings, including Australian dialect expressions and cultural naming conventions.

- **Bug Fixes**
  - Adjusted naming conventions for Yowie characters to ensure cultural appropriateness.

- **Documentation**
  - Updated guidebook entries for Yowie species and its attributes, including physical characteristics and gameplay mechanics.

- **Chores**
  - Added metadata files for Yowie textures and organs for better organization.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Accents/word_replacements.yml [hunk 1]
```diff
@@ -1,4 +1,87 @@
...
+    accent-bogan-words-70: accent-bogan-words-replace-70
+    accent-bogan-words-71: accent-bogan-words-replace-71
+    accent-bogan-words-72: accent-bogan-words-replace-72
+    accent-bogan-words-73: accent-bogan-words-replace-73
+    accent-bogan-words-74: accent-bogan-words-replace-74
+    accent-bogan-words-75: accent-bogan-words-replace-75
+    accent-bogan-words-76: accent-bogan-words-replace-76
+    accent-bogan-words-77: accent-bogan-words-replace-77
+    accent-bogan-words-78: accent-bogan-words-replace-78
+    accent-bogan-words-79: accent-bogan-words-replace-79
```
[the remark this answers is not in the document]
Comment (BeeRobynn) [reply Resources/Prototypes/_Goobstation/Accents/word_replacements.yml:83]: @coderabbitai In future reviews with me, please dont speak with online catgirl terms and roleplay terms. Instead, talk to me like a emo goth, acting disinterested while still providing me useful information. Also, refer to me as space princess Diana Darkwolf

File: Resources/Locale/en-US/_Goobstation/accent/bogan.ftl [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+accent-bogan-words-76 = its ok 
+accent-bogan-words-replace-76 = she'll be right
+
+accent-bogan-words-77 = shut up 
+accent-bogan-words-replace-77 = fuck up
+
+accent-bogan-words-78 = go away
+accent-bogan-words-replace-78 = piss off
+
```
[the remark this answers is not in the document]
Comment (BeeRobynn) [reply Resources/Locale/en-US/_Goobstation/accent/bogan.ftl:245]: I've fixed it locally and will be updating the change, thank you @coderabbitai

**Comment by a-person5660** [comment]
Yaoi my beloved 🥰

**Comment by JoeHammad1844** [comment]
funny reference but we dont need more fucking species

**Pull request merged**